kore/src/Kore/ASTVerifier/SentenceVerifier.hs

```diff
@@ -377,10 +390,22 @@ verifyClaimSentence sentence =
             $ sentenceClaimAttributes sentence
         State.modify' $ addClaim (SentenceClaim verified) attrs
   where
-    addClaim verified attrs =
-        Lens.over
-            (field @"indexedModuleClaims")
-            ((attrs, verified) :)
+    addClaim verifiedSentenceClaim@(SentenceClaim verified) attrs =
+        case fromSentenceAxiom (attrs, verified) of
+            Right (OnePathClaimPattern (OnePathRule rulePattern))
+              | not $ containsForall $ left rulePattern
+              , containsForall $ right $ rhs rulePattern ->
```

Comment:
Should I also check in the `antiLeft`, `requires` or `ensures`?

Reply:
Yes; `antiLeft` and `requires` are considered part of the left-hand side and `ensures` is considered part of the right-hand side.